app/controllers/application_controller.rb

```diff
@@ -23,6 +23,11 @@ def current_user_is_spo?
     sso_identity.current_user_is_spo?
   end
 
+  def service_notifications
```

Comment:
IMHO this should follow the pattern for PrisonsApplicationController#load_staff_id making it a before action rather than a helper method